src/scss/foundations/hiding.scss

```diff
@@ -0,0 +1,67 @@
+////
+/// Hiding Rules define all rules for hiding content based on device width.
+/// @group HIDING
+////
+
+@use "../lib";
+@use "breakpoints";
+
+/// Hide an element from `$fromBreakpoint` to `$untilBreakpoint` value.
+/// Outside the indicated breakpoints the element will have the assigned display value.
+///
+/// @param {String} $fromBreakpoint - Name of start breakpoint
+/// @param {String} $untilBreakpoint - Name of end breakpoint
+/// @throw {Error}
+///
+/// @example scss - Hide Mixin
+///  // Hide the element for every breakpoints
+///  .element {
+///     @include hide();
+///  }
+///
+///  // Hide element from breakpoint medium
+///  // Until breakpoint medium, the element 
+///  // display property will be `grid`
+///  .element {
+///     display: grid;
+///     @include hide(medium);
+///  }
+///
+///  // Hide element between breakpoint small and large
+///  // Until breakpoint medium, the element 
```

Comment:
```suggestion
///  // From breakpoint medium, the element 
```

Reply:
Su suggerimento di @emanuelemacri ho eliminato la seconda parte del commento in quanto potrebbe generare confusione. Il senso era che l'elemento viene nascosto a partire dal breakpoint medium, quindi "fino al" breakpoint medium avrebbe avuto display grid.